rules/windows/file/file_event/file_event_win_susp_winsxs_binary_creation.yml

```diff
@@ -0,0 +1,29 @@
+title: WinSxS Executable File Creation By Non-System Process
+id: 34746e8c-5fb8-415a-b135-0abc167e912a
+related:
+    - id: 64827580-e4c3-4c64-97eb-c72325d45399
+      type: derived
```

Comment:
I cannot find this rule/uuid

Reply:
Its in this PR the filename is `file_event_win_malware_snake_werfault_creation`

Reply:
xD